### PR TITLE
Introduction of new node based community edition to ensure cross-platform support

### PR DESCRIPTION
**Pull request by hrithikwins**
Introduction of new node based community edition to ensure cross-platform support
- Thanks to geng for making the original script robust
- Thanks to Michael to writing the documentation and ensuring understandability and clarity
- Thanks to Imaginer for bringing up the initiative to introduce cross platform support

reference: https://github.com/hrithikwins/ce-npm

Long Live Hubs! 😸

### CODE REVIEW

File: community-edition/readme.md [hunk 1]
```diff
@@ -17,36 +17,34 @@ Community Edition is designed for developers capable of working with the full Hu
 ## Prerequisites
 
 Before deploying the charts in this repo, you will need to choose and configure the following services...
```
Comment (Exairnous) [inline community-edition/readme.md:19]: This line (Before deploying the charts...) should be moved to after the Node.js instructions.

File: community-edition/readme.md [hunk 2]
```diff
@@ -17,36 +17,34 @@ Community Edition is designed for developers capable of working with the full Hu
...
+- Node.js installed on your system. You can download it from [here](https://nodejs.org/).
+- Clone the repository:
+   ```sh
+   git clone https://github.com/hubs-foundation/hubs-cloud
+   ```
+- Navigate to the project directory:
+   ```sh
+   cd hubs-cloud/community-edition
+   ```
+- Install module dependencies:
```
Comment (Exairnous) [inline community-edition/readme.md:29]:
```suggestion
- Install the module dependencies:
```

File: community-edition/readme.md [hunk 3]
```diff
@@ -17,36 +17,34 @@ Community Edition is designed for developers capable of working with the full Hu
...
-once the container has run, you should see a new file (hcce.yaml) in your COMMUNITY_EDITION_PATH
 
 ## Deploy to Kubernetes
 
 To deploy to your K8s cluster on your chosen hosting solution, follow these steps:
 
-- Add your chosen services into `render_hcce.sh`
-- Run `bash render_hcce.sh && kubectl apply -f hcce.yaml`
+- Add your chosen values into `input-values.yaml`
+- Run `npm run gen-hcce`
```
Comment (Exairnous) [inline community-edition/readme.md:47]: This still needs the kubectl apply command.
```suggestion
- Run `npm run gen-hcce && kubectl apply -f hcce.yaml`
```

**Review by Exairnous — CHANGES_REQUESTED** [review]
Thanks for this Hrithik!  Generally it looks pretty good, but I've left a couple inline comments and there are a couple other things that should be done as well:
1. You should link to the ce-npm repository where this was developed in the description of this PR so that people in the future can go to that repository for more details on how this was developed and the decisions that were made.
2. What are the `.DS_Store` files?  I don't think they should be added.
3. The `cbb.sh` file should be removed as it's no longer necessary.
4. The `cbb.yam` file should be removed from the main folder (it should only be in the `ssl_script` folder now).
5. The Dockerfile should be removed.
6. The `hcce.yam` file should be removed from the main folder (it should only be in the `generate_script` folder now).
7. The `render_hcce.sh` file should be removed as it's no longer necessary.

Don't worry about removing the bash scripts, I will branch them off into a separate branch before merging your PR.

File: community-edition/readme.md (hunk 1 above)
Comment (hrithikwins) [reply community-edition/readme.md]: Assuming that the user is following the nodejs script and therefore is preparing to get the kubernetes configuration file (usually .yaml ) file which can be applied to the cluster

**Review by hrithikwins — COMMENTED** [review]
LGTM, with the final changes the unnecessary files are removed and the necessary steps are added in the community-edition/readme.md

**Review by Exairnous — APPROVED** [review]
LGTM.  Thanks!

**Pull request merged**